Debug build is broken: 'plan' undeclared

Running `meson --buildtype=debug build` then `ninja -C build` shows a broken build, even though the release build works fine.
I just ran this on master, commit 8eb614f69e. 

```
[6/335] Compiling C object subprojects/cava-0.8.5/libcava.so.p/src_output_common.c.o
FAILED: subprojects/cava-0.8.5/libcava.so.p/src_output_common.c.o 
cc -Isubprojects/cava-0.8.5/libcava.so.p -Isubprojects/cava-0.8.5 -I../subprojects/cava-0.8.5 -I../subprojects/cava-0.8.5/include -I/usr/include/pipewire-0.3 -I/usr/include/spa-0.2 -I/usr/include/SDL2 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -O0 -g '-DPACKAGE="cava"' '-DVERSION="0.8.5"' '-DFONTDIR="/usr/local/share/consolefonts"' -DHAVE_ALLOCA_H -DNCURSES -DALSA -DPORTAUDIO -DPULSE -DSNDIO -DPIPEWIRE -DSDL -DSDL_GLSL -fPIC -D_REENTRANT -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -pthread -Wall -Wextra -Wno-unused-result -Wno-maybe-uninitialized -Wno-vla-parameter -MD -MQ subprojects/cava-0.8.5/libcava.so.p/src_output_common.c.o -MF subprojects/cava-0.8.5/libcava.so.p/src_output_common.c.o.d -o subprojects/cava-0.8.5/libcava.so.p/src_output_common.c.o -c ../subprojects/cava-0.8.5/src/output/common.c
In file included from ../subprojects/cava-0.8.5/src/output/common.c:3:
../subprojects/cava-0.8.5/src/output/common.c: In function ‘monstercat_filter’:
../subprojects/cava-0.8.5/include/cava/util.h:5:5: warning: ISO C forbids braced-groups within expressions [-Wpedantic]
    5 |     ({                                                                                             \
      |     ^
../subprojects/cava-0.8.5/src/output/common.c:18:29: note: in expansion of macro ‘max’
   18 |                 bars[m_y] = max(bars[z] - pow(de, 2), bars[m_y]);
      |                             ^~~
../subprojects/cava-0.8.5/include/cava/util.h:5:5: warning: ISO C forbids braced-groups within expressions [-Wpedantic]
    5 |     ({                                                                                             \
      |     ^
../subprojects/cava-0.8.5/src/output/common.c:22:29: note: in expansion of macro ‘max’
   22 |                 bars[m_y] = max(bars[z] - pow(de, 2), bars[m_y]);
      |                             ^~~
../subprojects/cava-0.8.5/include/cava/util.h:5:5: warning: ISO C forbids braced-groups within expressions [-Wpedantic]
    5 |     ({                                                                                             \
      |     ^
../subprojects/cava-0.8.5/src/output/common.c:30:29: note: in expansion of macro ‘max’
   30 |                 bars[m_y] = max(bars[z] / pow(monstercat * 1.5, de), bars[m_y]);
      |                             ^~~
../subprojects/cava-0.8.5/include/cava/util.h:5:5: warning: ISO C forbids braced-groups within expressions [-Wpedantic]
    5 |     ({                                                                                             \
      |     ^
../subprojects/cava-0.8.5/src/output/common.c:34:29: note: in expansion of macro ‘max’
   34 |                 bars[m_y] = max(bars[z] / pow(monstercat * 1.5, de), bars[m_y]);
      |                             ^~~
../subprojects/cava-0.8.5/src/output/common.c: In function ‘audio_raw_fetch’:
../subprojects/cava-0.8.5/src/output/common.c:431:73: error: ‘plan’ undeclared (first use in this function)
  431 |         mvprintw(n, 0, "%d: f:%f->%f (%d->%d), eq:%15e, peak:%d \n", n, plan->cut_off_frequency[n],
      |                                                                         ^~~~
```

--- a/include/modules/cava.hpp
+++ b/include/modules/cava.hpp
@@ -3,8 +3,10 @@
 #include "ALabel.hpp"
 #include "util/sleeper_thread.hpp"
 
+namespace cava {
 extern "C" {
 #include <cava/common.h>
+}
 }
 
 namespace waybar::modules {
@@ -21,13 +23,13 @@ class Cava final : public ALabel {
   util::SleeperThread thread_;
   util::SleeperThread thread_fetch_input_;
 
-  struct error_s error_ {};          // cava errors
-  struct config_params prm_ {};      // cava parameters
-  struct audio_raw audio_raw_ {};    // cava handled raw audio data(is based on audio_data)
-  struct audio_data audio_data_ {};  // cava audio data
-  struct cava_plan* plan_;           //{new cava_plan{}};
+  struct cava::error_s error_ {};          // cava errors
+  struct cava::config_params prm_ {};      // cava parameters
+  struct cava::audio_raw audio_raw_ {};    // cava handled raw audio data(is based on audio_data)
+  struct cava::audio_data audio_data_ {};  // cava audio data
+  struct cava::cava_plan* plan_;           //{new cava_plan{}};
   // Cava API to read audio source
-  ptr input_source_;
+  cava::ptr input_source_;
   // Delay to handle audio source
   std::chrono::milliseconds frame_time_milsec_{1s};
   // Text to display

--- a/meson.build
+++ b/meson.build
@@ -346,7 +346,7 @@ if get_option('experimental')
 endif
 
 cava = dependency('cava',
-                  version : '>=0.8.5',
+                  version : '>=0.9.1',
                   required: get_option('cava'),
                   fallback : ['cava', 'cava_dep'],
                   not_found_message: 'cava is not found. Building waybar without cava')

--- a/src/modules/cava.cpp
+++ b/src/modules/cava.cpp
@@ -155,7 +155,8 @@ auto waybar::modules::Cava::update() -> void {
     downThreadDelay(frame_time_milsec_, suspend_silence_delay_);
     // Process: execute cava
     pthread_mutex_lock(&audio_data_.lock);
-    cava::cava_execute(audio_data_.cava_in, audio_data_.samples_counter, audio_raw_.cava_out, plan_);
+    cava::cava_execute(audio_data_.cava_in, audio_data_.samples_counter, audio_raw_.cava_out,
+                       plan_);
     if (audio_data_.samples_counter > 0) audio_data_.samples_counter = 0;
     pthread_mutex_unlock(&audio_data_.lock);
 

--- a/src/modules/cava.cpp
+++ b/src/modules/cava.cpp
@@ -25,7 +25,7 @@ waybar::modules::Cava::Cava(const std::string& id, const Json::Value& config)
 
   // Override cava parameters by the user config
   prm_.inAtty = 0;
-  prm_.output = output_method::OUTPUT_RAW;
+  prm_.output = cava::output_method::OUTPUT_RAW;
   strcpy(prm_.data_format, "ascii");
   strcpy(prm_.raw_target, "/dev/stdout");
   prm_.ascii_range = config_["format-icons"].size() - 1;
@@ -34,9 +34,9 @@ waybar::modules::Cava::Cava(const std::string& id, const Json::Value& config)
   prm_.bar_spacing = 0;
   prm_.bar_height = 32;
   prm_.bar_width = 1;
-  prm_.orientation = ORIENT_TOP;
-  prm_.xaxis = xaxis_scale::NONE;
-  prm_.mono_opt = AVERAGE;
+  prm_.orientation = cava::ORIENT_TOP;
+  prm_.xaxis = cava::xaxis_scale::NONE;
+  prm_.mono_opt = cava::AVERAGE;
   prm_.autobars = 0;
   prm_.gravity = 0;
   prm_.integral = 1;
@@ -51,7 +51,7 @@ waybar::modules::Cava::Cava(const std::string& id, const Json::Value& config)
     prm_.upper_cut_off = config_["higher_cutoff_freq"].asLargestInt();
   if (config_["sleep_timer"].isInt()) prm_.sleep_timer = config_["sleep_timer"].asInt();
   if (config_["method"].isString())
-    prm_.input = input_method_by_name(config_["method"].asString().c_str());
+    prm_.input = cava::input_method_by_name(config_["method"].asString().c_str());
   if (config_["source"].isString()) prm_.audio_source = config_["source"].asString().data();
   if (config_["sample_rate"].isNumeric()) prm_.fifoSample = config_["sample_rate"].asLargestInt();
   if (config_["sample_bits"].isInt()) prm_.fifoSampleBits = config_["sample_bits"].asInt();
@@ -65,7 +65,7 @@ waybar::modules::Cava::Cava(const std::string& id, const Json::Value& config)
   if (config_["input_delay"].isInt())
     fetch_input_delay_ = std::chrono::seconds(config_["input_delay"].asInt());
   // Make cava parameters configuration
-  plan_ = new cava_plan{};
+  plan_ = new cava::cava_plan{};
 
   audio_raw_.height = prm_.ascii_range;
   audio_data_.format = -1;
@@ -155,12 +155,12 @@ auto waybar::modules::Cava::update() -> void {
     downThreadDelay(frame_time_milsec_, suspend_silence_delay_);
     // Process: execute cava
     pthread_mutex_lock(&audio_data_.lock);
-    cava_execute(audio_data_.cava_in, audio_data_.samples_counter, audio_raw_.cava_out, plan_);
+    cava::cava_execute(audio_data_.cava_in, audio_data_.samples_counter, audio_raw_.cava_out, plan_);
     if (audio_data_.samples_counter > 0) audio_data_.samples_counter = 0;
     pthread_mutex_unlock(&audio_data_.lock);
 
     // Do transformation under raw data
-    audio_raw_fetch(&audio_raw_, &prm_, &rePaint_);
+    audio_raw_fetch(&audio_raw_, &prm_, &rePaint_, plan_);
 
     if (rePaint_ == 1) {
       text_.clear();

--- a/subprojects/cava.wrap
+++ b/subprojects/cava.wrap
@@ -1,7 +1,7 @@
 [wrap-file]
-directory = cava-0.8.5
-source_url = https://github.com/LukashonakV/cava/archive/0.8.5.tar.gz
-source_filename = cava-0.8.5.tar.gz
-source_hash = 9ce3df7d374dc83ed0704fe3caef5e00600ce061d85608aad4142d2c59aa4647
+directory = cava-0.9.1
+source_url = https://github.com/LukashonakV/cava/archive/0.9.1.tar.gz
+source_filename = cava-0.9.1.tar.gz
+source_hash = 4df540b7f4892f72e48772929a15bc9ad61e2bce7a084be2df01c72ca5c02333
 [provide]
 cava = cava_dep

